mysql/hatch.toml

```diff
@@ -36,6 +36,13 @@ version = [
   "11.4",  # EOL 29 May 2029 (LTS version)
 ]
 
+[[envs.default.matrix]]
+python = ["3.12"]
+flavor = ["percona"]
+version = [
+  "8.0.34",
```

Comment:
Do we want a more recent version than this? The latest patch for 8.0 is 8.0.42. There's also 8.4.5 which is the latest